dashboard/assets/components/Dashboard.jsx

```diff
@@ -83,8 +83,9 @@ const appender = <T>(limit: number, mapper = replacer) => (update: Array<T>, pre
 // the execution of unnecessary operations (e.g. copy of the log array).
 const defaultContent: () => Content = () => ({
 	general: {
-		version: null,
-		commit:  null,
+		version:  null,
+		commit:   null,
+		syncMode: '',
```

Comment:
Pls remove the sync mode, that's mostly an internal detail, is not that useful on the outside, at least not now.